source/cloud/aws/sagemaker.md

```diff
@@ -58,6 +58,42 @@ Then in Jupyter select the `rapids` kernel when working with a new notebook.
 
 RAPIDS can also be used in [SageMaker Estimators](https://sagemaker.readthedocs.io/en/stable/api/training/estimators.html). Estimators allow you to launch training jobs on ephemeral VMs which SageMaker manages for you. The benefit of this is that your Notebook Instance doesn't need to have a GPU, so you are only charged for GPU instances for the time that your training job is running.
 
+All you’ll need to do is bring in your RAPIDS training script and libraries as a Docker container image and ask Amazon SageMaker to run copies of it in-parallel on a specified number of GPU instances in the cloud. Let’s take a closer look at how this works through a step-by-step approach:
+
+- Training script should accept hyperparameters as arguments. Starting with the base Rapids container (pulled from [Docker Hub](https://hub.docker.com/u/rapidsai)), use a Docker file to augment it by copying your training code and set `WORKDIR` path to the code.
+
+- Install [sagemaker-training toolkit](https://github.com/aws/sagemaker-training-toolkit) to make the container compatible with Sagemaker. Add other packages as needed for your workflow needs e.g. python, flask (model serving), dask-ml etc.
+
+- Push the image to a container registry (ECR).
+
+![Screenshot of summarized step to build Estimator](../../images/sagemaker-containerize-and-publish.png)
```

Comment:
That image mentioned the `0.15` nightly which is pretty old now. Could we update this image with some newer details? 

_I often file [Pixlr E](https://pixlr.com/e/) a quick tool for doing this._